src/services/jsDoc.ts

```diff
@@ -255,19 +259,21 @@ namespace ts.JsDoc {
     }
     function getCommentOwnerInfo(tokenAtPos: Node): CommentOwnerInfo | undefined {
         // TODO: add support for:
-        // - enums/enum members
-        // - interfaces
-        // - property declarations
         // - potentially property assignments

```

Comment:
do not think this is comment is needed either. we handle these in `BinaryExpression` section.